setup.py

```diff
@@ -2,10 +2,15 @@
 from os import path
 from setuptools import setup, find_packages
 
+with open(path.join(path.dirname(__file__), "README.md")) as f:
```

Comment:
Please use `open` with `encoding='utf-8'` (or `io.open`, if you wish to support Python 2) to prevent the following crash during the installation:
https://github.com/RGF-team/rgf/issues/65
https://github.com/henry0312/pytest-codestyle/pull/34